src/shared/Table/ResponsiveTable.tsx

```diff
@@ -318,7 +331,13 @@ export const DeletableRow: FunctionComponent<DeletableRowProps> = memo(
     return (
       <Tr
```

Comment:
Do you need `key` set on these?  I feel like this might be needed.

Reply:
The other thing is if that's the case, would it make sense to also expose a customizer function for these keys?